.github/workflows/release.yml

```diff
@@ -194,58 +189,109 @@ jobs:
           else
             echo "Server terminated successfully"
           fi
-      - name: Publish to PyPI
+      # PyPI publishing moved to after cross-platform testing
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist-main
+          path: dist
+
+  test-cross-platform:
+    name: Test Cross-Platform Installation
+    if: inputs.release_package_base == true || inputs.release_package_main == true
+    needs: [release-base, release-main]
```

Comment:
Should this be updated?

Reply:
i don't think so... `release_base` and `release_main` steps are above this.

This step is injected between the two.  I'll update the comment on the PR.